SpriterDotNet/Spriter.cs

```diff
@@ -29,22 +29,16 @@ public class SpriterFolder : SpriterElement
     public class SpriterFile : SpriterElement
     {
         [XmlAttribute("pivot_x")]
-        public float PivotX;
+        public float PivotX = 0f;
```

Comment:
I also prefer using inline initialisers but everything here is a big compromise to make most serialisers work. The thing is, some serialisers (like the built-in DataContractJsonSerializer) don't call the constructor at all (and initialiser == constructor in compiled IL). So I left the initialisation in the constructor but plan to move it in a separate Init method that will be called from the constructor or explicitly from deserialisers that don't call the constructor.